doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_cpu.rst

```diff
@@ -142,7 +143,10 @@ The Secure Domain has its own CPU, local RAM, and local peripherals to provide b
 The Secure Domain provides the initial root of trust (RoT), handles all the global resource allocation, acts as a trusted third party (TTP) between other MCU domains, and is used to secure cryptographic operations.
 Since the nRF54H platform supports global resource sharing, where memory partitions and peripherals in the global domain can be assigned to different local domains, the Secure Domain Firmware controls this partitioning while also acting as the boot master for the entire system.
 
-The Secure Domain Firmware (SDFW) exposes security-related services to the Cores in the system located in local domains (like Application and Radio).
+The Secure Domain Firmware (SDFW) is part of the nRF54H20 SoC binaries.
+It runs on the Secure Domain and exposes security-related services to the Cores in the system located in local domains (like Application and Radio).
+
+For more information on SDFW and the nRF54H20 SoC binaries, see :ref:`abi_compatibility` and :ref:`ug_nrf54h20_ironside`.
```

Comment:
Is the idea to use "SDFW" as a generic term for any firmware that runs in the secure domain? Should we clarify the distinction between IronSide SE and "SDFW" then?

Reply:
removed mentions to sdfw